Sources/IdentityProperties.swift

```diff
@@ -213,7 +213,15 @@ struct IdentityProperties: Codable {
         for reservedNamespace in IdentityProperties.reservedNamespaces {
             for namespace in identifiersMap.namespaces where namespace.caseInsensitiveCompare(reservedNamespace) == .orderedSame {
                 if let items = identifiersMap.getItems(withNamespace: namespace) {
-                    Log.debug(label: IdentityConstants.LOG_TAG, "IdentityProperties - Adding/Updating identifiers in namespace '\(namespace)' is not allowed.")
+                    if [IdentityConstants.Namespaces.IDFA, IdentityConstants.Namespaces.GAID].contains(namespace) {
+                        let logMessage = """
+                        IdentityProperties - Adding/Updating identifiers in namespace ‘\(namespace)’ is
+                        not allowed through this API; use MobileCore.setAdvertisingIdentifier instead.
+                        """
+                        Log.debug(label: IdentityConstants.LOG_TAG, logMessage)
```

Comment:
Updated log to warning level
One liner message is not possible with current string because max char limit of 200 (current 201). Changing "Adding/Updating" -> "Modifying" brings it down to 195 but triggers line length violation warning (line should be 180 or less)